edx_rest_framework_extensions/auth/jwt/tests/test_middleware.py

```diff
@@ -480,6 +499,14 @@ def test_set_request_user_with_use_jwt_cookie(
                 mock_log.warn.assert_not_called()
 
 
+# We want to duplicate these tests for now while we have two major code paths.  It will get unified once we have a
+# single way of doing JWT authentication again.
```

Comment:
For this duplicate class, I think I see the same issues detailed in my other comment.

Reply:
Feanil explained elsewhere. Marking as resolved.